tests/kafkatest/services/kafka/config.py

```diff
@@ -24,6 +24,11 @@ class KafkaConfig(dict):
     DEFAULTS = {
         config_property.SOCKET_RECEIVE_BUFFER_BYTES: 65536,
         config_property.LOG_DIRS: "/mnt/kafka/kafka-data-logs-1,/mnt/kafka/kafka-data-logs-2",
+        config_property.METADATA_LOG_DIR: "/mnt/kafka/kafka-metadata-logs",
+        config_property.METADATA_LOG_SEGMENT_BYTES: 8388608,
+        config_property.METADATA_LOG_BYTES_BETWEEN_SNAPSHOTS: 100,
```

Comment:
Hmm, I'm a bit confused about why we'd make something like "100 bytes between snapshots" the default. I guess the intention is to do a stress test?